kyaml/starlark/doc.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package starlark contains a kio.Filter which can be applied to resources to transform
```

Comment:
Probably it also worth to talk about why starlark is picked as a native supported language.

Reply:
Done